packages/flutter/lib/src/animation/animation_controller.dart

```diff
@@ -170,6 +197,18 @@ class AnimationController extends Animation<double>
   /// identifying animation controller instances in debug output.
   final String debugLabel;
 
+  /// The behavior of the animation when [AccessibilityFeatures.disableAnimations]
```

Comment:
the animation => the controller

Same as the last "See also" line

Reply:
Fixed